cmd/readmevalidation/coderresources.go

```diff
@@ -105,16 +108,11 @@ func validateCoderResourceTags(tags []string) error {
 	return nil
 }
 
-// Todo: This is a holdover from the validation logic used by the Coder Modules
-// repo. It gives us some assurance, but realistically, we probably want to
-// parse any Terraform code snippets, and make some deeper guarantees about how
-// it's structured. Just validating whether it *can* be parsed as Terraform
-// would be a big improvement.
-var terraformVersionRe = regexp.MustCompile("^\\s*\\bversion\\s+=")
-
 func validateCoderResourceReadmeBody(body string) []error {
-	trimmed := strings.TrimSpace(body)
 	var errs []error
+
+	trimmed := strings.TrimSpace(body)
+	// TODO: this may cause unexpected behaviour since the errors slice may have a 0 length. Add a test.
```

Comment:
can remove this comment, under the hood the go runtime will handle allocating a slice of length 1 if the length at the time of an append is 0